### DISCUSSION TITLE
what is the best programming language for mobile development?

### DISCUSSION THREAD

**Discussion by 8-Pac** [community/community#131791] [Programming Help]
### Body

So i just starting a new environment in mobile, but i still dont know what to choose for the programming language. any suggestion?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by LaluIman** [comment]
The best programming language for mobile development largely depends on the type of mobile application you want to develop, the platforms you're targeting (iOS, Android, or both), and your personal or team's expertise. Here are some popular languages and frameworks for mobile development: Swift(iOS), Kotlin(Android), Flutter(both devices), React Native(both devices), Kotlin Multiplatform(both device), When picking a language, think about what devices you're building for, how fast you need to develop, what skills you or your team already have, and how easy it will be to maintain your app in the future. Each language has its own perks and community, so choose the one that fits your project best.
